microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/core/ls/MicroProfileDelegateCommandHandlerForJava.java

```diff
@@ -68,6 +77,9 @@ public class MicroProfileDelegateCommandHandlerForJava extends AbstractMicroProf
 	private static final String JAVA_DEFINITION_COMMAND_ID = "microprofile/java/definition";
 	private static final String JAVA_DIAGNOSTICS_COMMAND_ID = "microprofile/java/diagnostics";
 	private static final String JAVA_HOVER_COMMAND_ID = "microprofile/java/hover";
+	private static final String JAVA_WORKSPACE_SYMBOLS_ID = "microprofile/java/workspaceSymbols";
+	
+	private final IJavaWorkspaceSymbolsParticipant symbolsParticipant = new JaxRsWorkspaceSymbolParticipant();
```

Comment:
To be consistent with another participant, it should be better to provide an extension point.